rad/src/operators/float.rs

```diff
@@ -42,3 +59,183 @@ pub fn less_than(input: &RadonFloat, args: &[Value]) -> Result<RadonBoolean, Rad
+    Ok(RadonFloat::from(input.value() % modulo))
+}
+
+pub fn negate(input: &RadonFloat) -> RadonFloat {
+    RadonFloat::from(-input.value())
+}
+
+pub fn power(input: &RadonFloat, args: &[Value]) -> Result<RadonFloat, RadError> {
+    let wrong_args = || RadError::WrongArguments {
+        input_type: RadonFloat::radon_type_name(),
+        operator: "Power".to_string(),
+        args: args.to_vec(),
+    };
+
+    let arg = args.first().ok_or_else(wrong_args)?.to_owned();
+    let exp = from_value::<i32>(arg).map_err(|_| wrong_args())?;
+
+    Ok(RadonFloat::from(input.value().powi(exp)))
+}
+
+pub fn floor(input: &RadonFloat) -> RadonFloat {
+    RadonFloat::from(input.value().floor())
+}
+
+pub fn round(input: &RadonFloat) -> RadonFloat {
+    RadonFloat::from(input.value().round())
+}
+
+pub fn truncate(input: &RadonFloat) -> RadonFloat {
+    RadonFloat::from(input.value().trunc())
+}
+
+#[test]
+fn test_float_absolute() {
+    let positive_integer = RadonFloat::from(10.0);
+    let negative_integer = RadonFloat::from(-10.0);
+
+    assert_eq!(absolute(&positive_integer), positive_integer);
+    assert_eq!(absolute(&negative_integer), positive_integer);
+}
+
+#[test]
+fn test_float_to_string() {
+    let rad_int = RadonFloat::from(10.2);
+    let rad_string: RadonString = RadonString::from("10.2");
+
+    assert_eq!(to_string(rad_int).unwrap(), rad_string);
+}
+
+#[test]
+fn test_float_multiply() {
+    let rad_int = RadonFloat::from(10.0);
+    let value = Value::Float(3.0);
+
+    assert_eq!(
+        multiply(&rad_int, &[value]).unwrap(),
+        RadonFloat::from(30.0)
+    );
+}
+
+#[test]
+fn test_float_greater() {
+    let rad_int = RadonFloat::from(10.0);
+    let value = Value::Float(9.9);
+    let value2 = Value::Float(10.0);
+    let value3 = Value::Float(10.1);
+
+    assert_eq!(
+        greater_than(&rad_int, &[value]).unwrap(),
+        RadonBoolean::from(true)
+    );
+    assert_eq!(
+        greater_than(&rad_int, &[value2]).unwrap(),
+        RadonBoolean::from(false)
+    );
+    assert_eq!(
+        greater_than(&rad_int, &[value3]).unwrap(),
+        RadonBoolean::from(false)
+    );
+}
+
+#[test]
+fn test_float_less() {
+    let rad_int = RadonFloat::from(10.0);
+    let value = Value::Float(9.9);
+    let value2 = Value::Float(10.0);
+    let value3 = Value::Float(10.1);
+
+    assert_eq!(
+        less_than(&rad_int, &[value]).unwrap(),
+        RadonBoolean::from(false)
+    );
+    assert_eq!(
+        less_than(&rad_int, &[value2]).unwrap(),
+        RadonBoolean::from(false)
+    );
+    assert_eq!(
+        less_than(&rad_int, &[value3]).unwrap(),
+        RadonBoolean::from(true)
+    );
+}
+
+#[test]
+fn test_float_negate() {
+    let positive_integer = RadonFloat::from(10.0);
+    let negative_integer = RadonFloat::from(-10.0);
+
+    assert_eq!(negate(&positive_integer), negative_integer);
+    assert_eq!(negate(&negative_integer), positive_integer);
+}
+
+#[test]
+fn test_float_modulo() {
```

Comment:
For the sake of documentation, could you also test the result of `(-1.0) % 2.0` here?